src/components/tables/Categories.tsx

```diff
@@ -1,78 +1,74 @@
 import { useMemo } from 'react';
 import { Row } from 'react-table';
-import { Category } from '../../model/category';
-import { ColumnDefinition, TableAction } from '../../model/table';
+import { ColumnDefinition, TableAction } from '../../model/table/table-definitions';
 import AdvancedTable, { AdvancedTableProps } from '../table/AdvancedTable';
+import { CategoryType } from '../../model/enums/category-type';
```

Comment:
Please change imports to absolute path (just from `/src` directory, so probably remove only `../../`).

Reply:
Fixed.